scripts/us_eia/opendata/process/nuclear.py

```diff
@@ -0,0 +1,145 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""EIA Nuclear Status Dataset specific functions."""
+
+import common
+import logging
+import re
+
+
+def extract_place_statvar(series_id, counters):
+    """Given the series_id, extract the raw place and stat-var ID.
+
+    Args:
+        series_id: EIA series ID
+        counters: map for updating error statistics
+
+    Returns a (place, raw-stat-var, is_us_place) tuple.
+    """
+    m = re.match(r"^NUC_STATUS\.([^.]+)\.([^.]+)\.(D)$", series_id)
+    if m:
+        measure = m.group(1)
+        place = m.group(2)
+        if not place == 'US':
+            place = f'dcid:eia/pp/{m.group(2)}'
```

Comment:
This probably cases double dcid: prefixes, given: https://github.com/datacommonsorg/data/pull/434/files#diff-378d6eed12778a313c7dfb5c32f67dc82177db8e114259d54e2a277060bcb871L358

Instead of prefixing "dcid:", perhaps use "eia/" prefix to identify if its an official place, or something "eia/" specific that we can ignore?